### ISSUE TITLE
Error: InternalError... when running on Apollo Package

### ISSUE THREAD

**Issue by possen** [unsignedapps/swift-create-xcframework#84]
I am trying to run the command on Apollo project https://github.com/apollographql/apollo-ios

I am running Xcode 13.4.1
swift-driver version: 1.45.2 Apple Swift version 5.6.1 (swiftlang-5.6.0.323.66 clang-1316.0.20.12)
Target: arm64-apple-macosx12.0

### Checklist
- [x] If possible, I've reproduced the issue using the `master` branch of this package
- [x] I've searched for [existing GitHub issues](https://github.com/unsignedapps/swift-create-xcframework/issues)

### Steps to Reproduce

I installed it using mint.  `mint install unsignedapps/swift-create-xcframework`

git clone https://github.com/apollographql/apollo-ios
cd apollo-ios
swift create-xcframework --platform ios --platform macos Apollo

### Expected behavior

expected to command to complete with XCFramework generated

### Actual behavior

```
swift create-xcframework --platform ios --platform macos Apollo

debug: evaluating manifest for 'apollo-ios' v. unknown 
debug: evaluating manifest for 'swift-argument-parser' v. 1.2.0 
debug: evaluating manifest for 'inflectorkit' v. 1.0.0 
debug: evaluating manifest for 'sqlite.swift' v. 0.13.3 
debug: evaluating manifest for 'swift-collections' v. 1.0.3 
debug: evaluating manifest for 'apollo-ios' v. unknown 
Error: InternalError(description: "Internal error. Please file a bug at https://bugs.swift.org with this info. plugin not supported")

swift create-xcframework --list-products            

debug: evaluating manifest for 'apollo-ios' v. unknown 
debug: evaluating manifest for 'inflectorkit' v. 1.0.0 
debug: evaluating manifest for 'swift-argument-parser' v. 1.2.0 
debug: evaluating manifest for 'sqlite.swift' v. 0.13.3 
debug: evaluating manifest for 'swift-collections' v. 1.0.3 
debug: evaluating manifest for 'apollo-ios' v. unknown 
Error: InternalError(description: "Internal error. Please file a bug at https://bugs.swift.org with this info. plugin not supported")
```

Tried on an Intel Mac with Xcode 14 as well
```
swift create-xcframework --platform ios --platform macos Apollo

debug: evaluating manifest for 'apollo-ios' v. unknown 
debug: evaluating manifest for 'apollo-ios' v. unknown 
debug: [DependencyResolver] derived 'swift-argument-parser' requirement '1.1.2..<2.0.0'
debug: [DependencyResolver] derived 'swift-collections' requirement '1.0.0..<2.0.0'
debug: [DependencyResolver] derived 'inflectorkit' requirement '1.0.0..<1.1.0'
debug: [DependencyResolver] derived 'sqlite.swift' requirement '0.13.1..<0.14.0'
debug: [DependencyResolver] resolving 'swift-argument-parser'
debug: evaluating manifest for 'swift-argument-parser' v. 1.1.3 
debug: [DependencyResolver] resolved 'swift-argument-parser' @ '1.1.3'
debug: [DependencyResolver] resolving 'swift-collections'
debug: evaluating manifest for 'swift-collections' v. 1.0.1 
debug: [DependencyResolver] resolved 'swift-collections' @ '1.0.1'
debug: [DependencyResolver] resolving 'inflectorkit'
debug: evaluating manifest for 'inflectorkit' v. 1.0.0 
debug: [DependencyResolver] resolved 'inflectorkit' @ '1.0.0'
debug: [DependencyResolver] resolving 'sqlite.swift'
debug: evaluating manifest for 'sqlite.swift' v. 0.13.2 
debug: [DependencyResolver] resolved 'sqlite.swift' @ '0.13.2'
debug: evaluating manifest for 'swift-argument-parser' v. 1.1.3 
debug: evaluating manifest for 'swift-collections' v. 1.0.1 
debug: evaluating manifest for 'inflectorkit' v. 1.0.0 
debug: evaluating manifest for 'sqlite.swift' v. 0.13.2 
debug: [DependencyResolver] solved 'inflectorkit' (https://github.com/mattt/InflectorKit) at '1.0.0'
debug: [DependencyResolver] solved 'sqlite.swift' (https://github.com/stephencelis/SQLite.swift.git) at '0.13.2'
debug: [DependencyResolver] solved 'swift-argument-parser' (https://github.com/apple/swift-argument-parser.git) at '1.1.3'
debug: [DependencyResolver] solved 'swift-collections' (https://github.com/apple/swift-collections) at '1.0.1'
debug: [DependencyResolver] solved 'apollo-ios' (/Users/possen/Projects/apollo-ios) at 'unversioned'
debug: [DependencyResolver] dependency resolution complete!
debug: adding 'sqlite.swift' (https://github.com/stephencelis/SQLite.swift.git) to managed dependencies
debug: adding 'swift-argument-parser' (https://github.com/apple/swift-argument-parser.git) to managed dependencies
debug: adding 'inflectorkit' (https://github.com/mattt/InflectorKit) to managed dependencies
debug: adding 'swift-collections' (https://github.com/apple/swift-collections) to managed dependencies
debug: evaluating manifest for 'swift-collections' v. 1.0.1 
debug: evaluating manifest for 'sqlite.swift' v. 0.13.2 
debug: evaluating manifest for 'swift-argument-parser' v. 1.1.3 
debug: evaluating manifest for 'inflectorkit' v. 1.0.0 
debug: evaluating manifest for 'apollo-ios' v. unknown 
Error: InternalError(description: "Internal error. Please file a bug at https://bugs.swift.org with this info. plugin not supported")
```
I tried on a different package and it worked correctly.

Any suggestions on what I can do?

**Comment by Lessica** [comment]
I'm facing this same issue trying to build https://github.com/apple/swift-argument-parser .

**Comment by Lessica** [comment]
Solved by `rm -f Package@swift-5.6.swift`.
